### PR TITLE
Updated for Naginator-Opt-Out property Implementation

### PR DESCRIPTION
**Pull request by erenatas**
Updated opt-out definition. Implementation now takes advantage of ```convert_mapping_to_xml``` implementation.

Signed-off-by: Eren ATAS <eatas.contractor@libertyglobal.com>

### CODE REVIEW

File: jenkins_jobs/modules/properties.py [hunk 1]
```diff
@@ -234,19 +234,21 @@ def naginator_opt_out(registry, xml_parent, data):
     """yaml: naginator-opt-out
     Lets you opt-out so no rebuild option for Naginator is added.
 
-
     Requires the Jenkins :jenkins-plugins:`Naginator Plugin <naginator>`.
 
+    :arg bool opt-out: enables the rebuild option (default False).
```
Comment (romanek-adam) [inline jenkins_jobs/modules/properties.py]: disables?

File: jenkins_jobs/modules/properties.py [hunk 2]
```diff
@@ -234,19 +234,21 @@ def naginator_opt_out(registry, xml_parent, data):
     """yaml: naginator-opt-out
```
Comment (romanek-adam) [inline jenkins_jobs/modules/properties.py:234]: Btw shouldn't this property be simply named "naginator"? I think that was the major point - we want to have some potential for future configuration of this plugin, right?

Comment (romanek-adam) [reply jenkins_jobs/modules/properties.py:234]: OK my bad, the XML element is named NaginatorOptOutProperty so the name of this property is fine.

File: jenkins_jobs/modules/properties.py (hunk 1 above)
Comment (romanek-adam) [reply jenkins_jobs/modules/properties.py]: I guess you should push the fix upstream too, right?

**Review by romanek-adam — APPROVED** [review]

**Review by vdanaila — APPROVED** [review]

**Pull request merged**